File: GolPredictor.Web/src/app/core/models/sesion-dto.ts
import { PartidoDto } from "./partido-dto";
import { SesionUsuarioDto } from "./sesion-usuario-dto";

export interface SesionDto {
    id: number;
    nombre: string;
    entryCode: string;
    partidoId: number | null;
    status: boolean | null;
    partido: PartidoDto;
    sesionUsuario: SesionUsuarioDto[];
}

File: GolPredictor.WebApi/GolPredictor.WebApi/DTO/Sesion/SesionDto.cs
using GolPredictor.WebApi.DataAccess.Entities;
using GolPredictor.WebApi.DTO.Partido;
using System.Collections.Generic;

namespace GolPredictor.WebApi.DTO.Sesion
{
    public class SesionDto
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string EntryCode { get; set; }
        public int? PartidoId { get; set; }
        public bool? Status { get; set; }
        public PartidoDto Partido { get; set; }
        public  ICollection<SesionUsuarioDto> SesionUsuario { get; set; }
    }
}
